packaging/python/Dockerfile_gpu

```diff
@@ -1,21 +1,15 @@
 FROM neuronsimulator/neuron_wheel
-LABEL authors="Pramod Kumbhar, Fernando Pereira, Alexandru Savulescu"
+LABEL authors="Pramod Kumbhar, Olli Lupton, Fernando Pereira, Alexandru Savulescu"
 
 WORKDIR /root
 
-# download nvhpc 21.2 rpms. Note that newer versions until at least 21.7 has various
-# bugs and hence we are sticking to 21.2 until we verify latest release.
-# see https://github.com/BlueBrain/CoreNeuron/issues/605
-RUN wget --no-verbose \
-      https://developer.download.nvidia.com/hpc-sdk/21.2/nvhpc-21-2-21.2-1.x86_64.rpm \
-      https://developer.download.nvidia.com/hpc-sdk/21.2/nvhpc-2021-21.2-1.x86_64.rpm \
-      https://developer.download.nvidia.com/hpc-sdk/21.2/nvhpc-21-2-cuda-multi-21.2-1.x86_64.rpm \
-    && yum install -y *.rpm \
-    && rm *.rpm && yum clean all
+# Install NVHPC from NVIDIA's repository.
+RUN yum-config-manager --add-repo https://developer.download.nvidia.com/hpc-sdk/rhel/nvhpc.repo \
```

Comment:
22.2 is the first release where they document doing it this way, but they seem to have added older version RPMs too so I figured it would be OK with 22.1 as well.